### ISSUE TITLE
Rewording

### ISSUE THREAD

**Issue by amandaqu18** [mbadalamente/Module6#8]
System.out.println("On a scale from 1-10, how much will you participate in class today?");

Maybe instead of saying how much will you participate in class today that can make the user reading think its a mean comment but maybe reword it to how much did you feel you learned today in class to see how active the user was during class.

**Comment by mbadalamente** [comment]
I think that the intention when creating this part was to enter the zoom preferences for the day to see if the user wanted to participate before class starts. It seems like its more of a prequiz than a postquiz feature

**Comment by ash730** [comment]
I actually really like this idea. You could even take it a step further and word it as something friendlier so it feels more personal so they can open up. Maybe something like. "Hey there, I hope your day is going well! Based on how you're feeling today on a scale of 1 to 10 what are the chances of participating in class?" or something like that.

**Comment by hutch445** [comment]
I like the idea of expanding what the code asks to make it friendlier although Madison does have a point. Maybe making the Outputs friendlier will lead to a better satisfaction rate with the program. Maybe make the user feel like its less of a quiz and more of a get to know how you are doing?
